src/dataprotection/azext_dataprotection/manual/Manifests/AzureBlob.py

```diff
@@ -13,6 +13,18 @@
   "allowedRestoreTargetTypes": [ "OriginalLocation" ],
   "itemLevelRecoveyEnabled": true,
   "supportSecretStoreAuthentication": false,
+  "backupVaultPermissions": [
+    {
+      "roleDefinitionName": "Storage Account Backup Contributor",
+      "assignedOn": "storageGroup",
```

Comment:
accessMethod is same for all the datasource-related permission making this field redundant. We can skip this field as there is no cross datasource type customization needed here.

Also looks like assignedOn does not cater to any actual function. I think that too can be skipped.